doc/Installation/Installation-Ubuntu-1604-Apache.md

```diff
@@ -98,9 +98,17 @@ Edit the text which says `RANDOMSTRINGGOESHERE` and set your own community strin
     chmod +x /usr/bin/distro
     systemctl restart snmpd
 
-### Cron job
+### Background jobs
 
-    cp /opt/librenms/librenms.nonroot.cron /etc/cron.d/librenms
+If your upgrading LibreNMS, ensure to remove the crontab:
+
+    rm -fv /etc/cron.d/librenms
+
+Copy, enable and start systemd timers:
+
+    cp /opt/librenms/contrib/systemd/* /lib/systemd/system/
+    systemctl daemon-reload
+    for timer in /lib/systemd/system/librenms@*.timer; do systemctl enable $(basename $timer) && systemctl start $(basename $timer); done
```

Comment:
You need to change lib to etc here too.
Also systemctl enable has a `--now` option which also starts the unit.

Reply:
Great, I didn't know the `--now` argument.